storage/understanding-ephemeral-storage.adoc

```diff
@@ -12,4 +12,6 @@ include::modules/storage-ephemeral-storage-types.adoc[leveloffset=+1]
 
 include::modules/storage-ephemeral-storage-manage.adoc[leveloffset=+1]
 
+For information about defining quotas for namespaces, see xref:../applications/quotas/quotas-setting-per-project.adoc[Quota setting per project].
```

Comment:
```suggestion
For information about defining quotas for projects, see xref:../applications/quotas/quotas-setting-per-project.adoc[Quota setting per project].
```
Hi @lpettyjo  ,maybe change `namespaces` -> `projects` is better. It'll keep the same with the reference link title. Actually they're the same scope. In addition, we'd better add the setting influence.
References:  https://kubernetes.io/blog/2022/09/19/local-storage-capacity-isolation-ga/
```
With this setting in pod spec, it will affect how the scheduler makes a decision on scheduling pods and also how kubelet evict pods.

First of all, the scheduler ensures that the sum of the resource requests of the scheduled containers is less than the capacity of the node. In this case, the pod can be assigned to a node only if its available ephemeral storage (allocatable resource) has more than 4GiB (request total).

Secondly, at container level, since one of the container sets resource limit, kubelet eviction manager will measure the disk usage of this container and evict the pod if the storage usage of the first container exceeds its limit (8GiB). At pod level, kubelet works out an overall Pod storage limit by adding up the limits of all the containers in that Pod. In this case, the total storage usage at pod level is the sum of the disk usage from all containers plus the Pod's emptyDir volumes. If this total usage exceeds the overall Pod storage limit (8GiB), then the kubelet also marks the Pod for eviction.

```
 Others LGTM!

Reply:
Yes agree may be good to add a line or two on how that feature works at the scheduler level.

Reply:
Done! Thanks for your feedback!